tests/external-links.js

```diff
@@ -40,7 +41,7 @@ try {
   const externalUrlSet = new Set();
 
   crawler.addEventListener(`externalLinkFound`, ({ url }) => {
-    if (!excludedUrls.some(excludedUrl => url.startsWith(excludedUrl))) {
+    if (excludedUrls.some(excludedUrl => url.startsWith(excludedUrl))) {
```

Comment:
Am I missing something or is this now wrong? Or was this only for testing?

Reply:
I tried to refactor the function to the following:
```ts
if (excludedUrls.some(excludedUrl => url.startsWith(excludedUrl))) return;
externalUrlSet.add(url);
process.stdout.write(`\r${externalUrlSet.size} link(s) found.`);
```

But eslint didn't like it so I put it back. I must have missed the `!`. Good spot 👍

Reply:
> eslint didn't like it

Probably because you didn't put braces (and newlines) around the `return` statement. Other than that, there should be nothing wrong with early returns. In fact, I actually do prefer them over long `if` bodies as well! But in this case, since it's only two lines, I guess it's okay the way it is.